# HG changeset patch
# User Nathan Phillip Brink <ohnobinki@ohnopublishing.net>
# Date 1293590270 18000
# Node ID 6d6117aa0da18aa3af0ca28f7e3537f304998f2c
# Parent  2d23479cb9e66efae0b5816ba68a827f7fd7d035
Use CFLAGS when using the compiler driver during the linking stage. Fixes Gentoo portage-multilib compilation failure.

diff -r 2d23479cb9e6 -r 6d6117aa0da1 Makefile
--- a/Makefile	Tue Dec 28 21:33:11 2010 -0500
+++ b/Makefile	Tue Dec 28 21:37:50 2010 -0500
@@ -144,7 +144,7 @@
 # mdadm.tcc doesn't work..
 
 mdadm : $(OBJS)
-	$(CC) $(LDFLAGS) -o mdadm $(OBJS) $(LDLIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o mdadm $(OBJS) $(LDLIBS)
 
 mdadm.static : $(OBJS) $(STATICOBJS)
 	$(CC) $(LDFLAGS) -static -o mdadm.static $(OBJS) $(STATICOBJS)
@@ -167,7 +167,7 @@
 
 # use '' to guarantee no dynamic linker interactions with the monitor thread
 mdmon : $(MON_OBJS)
-	$(CC) $(LDFLAGS) $(MON_LDFLAGS) -z now -o mdmon $(MON_OBJS) $(LDLIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(MON_LDFLAGS) -z now -o mdmon $(MON_OBJS) $(LDLIBS)
 msg.o: msg.c msg.h
 
 test_stripe : restripe.c mdadm.h
